### DISCUSSION TITLE
Role assignment for google auth

### DISCUSSION THREAD

**Discussion by alex88** [grafana/grafana#44456] [General]
It [seems](https://github.com/grafana/grafana/issues/28121#issuecomment-707390913) that role_attribute_path doesn't work with google auth, would it be possible to include that for all oauth implementations? Otherwise, is there a way to assign a role depending on the user that's logging in?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
